src/api.js

```diff
@@ -70,7 +69,7 @@ export default class Api {
      * The even emitter instance.
      * @type {EventEmitter}
      */
-    this.events = options.events || new EventEmitter();
+    this.events = options.events;
```

Comment:
As events is now a string requirements, we should probably throw when not provided — as we did for `KintoBase` and `Collection`. Also, we could consider this not being an *option* anymore, but a required argument; though this would be an important API change… Thoughts?